### DISCUSSION TITLE
Using sidekiq-pro / sidekiq-ent with older ruby versions

### DISCUSSION THREAD

**Discussion by felixcuello** [sidekiq/sidekiq#5125] [Q&A] [answered]
Hello,

I saw the requirements is ruby 2.5+ for running sidekiq-pro / sidekiq-ent

I was able to install it with ruby 2.4.10 in one of our systems by just injecting the gem... that worked. and I have two questions:

1. Is it going to work?  (I did some smoke tests and apparently worked)
2. I have to use sidekiq-pro / sidekiq-ent in one of our old containers which have ruby 1.9.3, do you think it's possible to do that?

Thanks

**Comment by mperham** [comment, marked as answer]
See the Pro and Ent major version upgrade notes in the root of the repo. They say which versions are required. I don't believe there's any version that supports Ruby 1.9 and I wouldn't support it anyways.
